Misc/NEWS.d/next/Documentation/2022-02-08-15-38-16.bpo-46586.6qVFVL.rst

```diff
@@ -0,0 +1 @@
+Fix enum.property link to point to the enum page rather than the built-in property() page
```

Comment:
Not sure this needs a NEWS entry at all.

Reply:
Yeah sorry about that, I used blurb-it for the first time and by the time I was done, I saw that the skip-news label had been added. Should I just make another commit to remove this file?

Reply:
Yes, please!